src/components/UserCard/index.tsx

```diff
@@ -32,8 +32,9 @@ const UserCard: React.FC<UserCardProps> = ({ user, isShort }) => {
     <Card>
       <UserAvatar src={user.avatarUrl} />
       <UserName>{user.name}</UserName>
-      <GhUserContainer>
-        <FaGithub size={20} /> <span>{user.ghUsername}</span>
+      <GhUserContainer href={`https://github.com/${user.ghUsername}`}>
```

Comment:
Como se trata de um link para uma página externa, seria interessante abrir o link em uma nova aba, mudando o target. [Não esquecer de adicionar o `noopener` e o `noreferrer`.](https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable)